docs/modules/releases/pages/v0.36.0.adoc

```diff
@@ -0,0 +1,84 @@
+include::ROOT:partial$attributes.adoc[]
+
+[#v0.36.0]
+= Cerbos v0.36.0
+
+== Highlights
+
+To reduce the overhead of writing large audit log entries to slow sinks (files and `stdout`, for example), Cerbos now writes audit logs in the background. If you send very large batch requests containing a lot of data to Cerbos, this should help improve the response times.
+
+A community contribution from @rcrowe makes the Kafka audit backend use system CA certificates if none are provided explicitly in configuration. It also addresses a case where asynchronous Kafka writes start blocking when the downstream brokers are down.
+
+The new xref:cli:cerbosctl.adoc#inspect-policies[`cerbosctl inspect`] command provides command-line access to the `inspect` Admin API endpoint introduced in the previous release. Currently it supports listing actions covered by each policy. More policy inspection options are planned for future releases.
+
+=== Cerbos Hub integration
+
+Early adopters of the link:https://www.cerbos.dev/product-cerbos-hub[Cerbos Hub] audit log collection feature can now filter out audit log entries locally before they are sent to Hub.
+
+For consistency, the `bundle` storage driver has been renamed to `hub`. To migrate, change `storage.driver: bundle` to `storage.driver: hub` and rename any configuration values starting with `storage.bundle` to `storage.hub`.
+
+Embedded PDP users can use the `cerbosctl hub epdp list-candidates` command to scan a policy repo and list the set of policies that would be included in a Cerbos Embedded PDP bundle.
+
+== Changelog
+
+
+=== Bug Fixes
+
+* Eagerly establish gRPC connection to avoid initial delay (link:https://github.com/cerbos/cerbos/pull/2105[#2105])
```

Comment:
```suggestion
* Default expectation to `EFFECT_DENY` for unspecified actions in tests (link:https://github.com/cerbos/cerbos/pull/2116[#2116])
* Eagerly establish gRPC connection to avoid initial delay (link:https://github.com/cerbos/cerbos/pull/2105[#2105])
```